service/controller.go

```diff
@@ -3130,7 +3140,8 @@ func (s *service) SelectOrCreateFCPGForHost(ctx context.Context, symID string, h
 			}
 			for _, initiatorID := range initList.InitiatorIDs {
 				_, dirPort, _, err := splitFibreChannelInitiatorID(initiatorID)
-				if err != nil {
+				// Filter SCSI_FC ports; skip the port if it is not a SCSI_FC port
+				if err != nil || !slices.Contains(scsiPorts, dirPort) {
 					continue
 				}
 				portListFromHost = appendIfMissing(portListFromHost, dirPort)
```

Comment:
Should '_isValidHost_' be set upon checking the length of _portListFromHost_ when there is a chance of all ports getting skipped?

Reply:
Thanks for pointing that out, Santosh. After reviewing, I realized that isValidHost isn't needed, so I've removed it.